android/java/org/chromium/chrome/browser/toolbar/top/BraveToolbarLayout.java

```diff
@@ -434,17 +434,24 @@ public void onClick(View v) {
     if (mBraveShieldsButton == v && mBraveShieldsButton != null) {
       showShieldsMenu(mBraveShieldsButton);
     } else if (mBraveRewardsButton == v && mBraveRewardsButton != null) {
-      Context context = getContext();
-      if (checkForRewardsOnboarding()) {
-        OnboardingPrefManager.getInstance().showOnboarding(context);
-        hideRewardsOnboardingIcon();
-      } else {
-        if (null != mRewardsPopup) {
-          return;
-        }
-        mRewardsPopup = new BraveRewardsPanelPopup(v);
-        mRewardsPopup.showLikePopDownMenu();
+      // Context context = getContext();
```

Comment:
@SergeyZhukovsky i was not about recently updated onboarding so i have asked for the confirmation. so till then i have commented out below block.,